app/helpers/submission_inputs_helper.rb

```diff
@@ -399,23 +399,22 @@ def generate_ontology_select_input(name, label, selected, multiple)
                          open_to_add: true)
   end
 
-  def generate_list_text_input(attr)
+  def generate_list_text_input(attr, helper_text: nil)
     label = attr_header_label(attr)
     values = attr.values || ['']
     name = attr.name
-    generate_list_field_input(attr, name, label, values) do |value, row_name, id|
-      text_input(label: '', name: row_name, value: value)
+    generate_list_field_input(attr, name, label, values, helper_text: helper_text) do |value, row_name, id|
+      text_area_tag(row_name, value, class: 'input-field-component', label: '')
```

Comment:
why this change ?

Reply:
It is to enlarge the 'change notes' section in order to write a long text
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/a06cf41e-3ea6-4784-a128-316c58dd8b85)

Reply:
yes, but this will make all the `text_input` with a big `text area`.
This change needs to affect only changes notes